mlx/ops.cpp

```diff
@@ -4101,6 +4101,15 @@ array number_of_elements(
       {a}));
 }
 
+array conjugate(const array& a, StreamOrDevice s /* = {} */) {
+  // Mirror NumPy's behaviour for real input
+  if (a.dtype() != complex64) {
+    return a;
+  }
```

Comment:
Ah I see this now, you can disregard previous comments. I think this is ok instead of doing the buffer copy.